src/components/ClauseComponent.js

```diff
@@ -36,6 +44,70 @@ function ClauseComponent(props) {
     iconBg: clauseProps.CLAUSE_BACKGROUND
   };
 
+  /* eslint react/no-find-dom-node: "off" */
+  useEffect(() => {
+    const findPosition = (key) => {
+      const pathToConditional = props.editor.value.document.getPath(key);
+      const elementDOM = props.editor.findDOMNode(pathToConditional);
+      const popupPosition = 'bottom left';
+      const positionalStyle = {
+        popupStyle: { top: elementDOM.offsetTop, left: elementDOM.offsetLeft, transform: 'none' },
+        popupPosition,
+      };
+      return positionalStyle;
+    };
+
+    const findConditionals = node => ({
+      ...(node.type === 'conditional' ? {
+        [node.key]: {
+          whenTrue: node.data.get('whenTrue'),
+          whenFalse: node.data.get('whenFalse'),
+          position: findPosition(node.key),
+          isFalse: node.text === '',
```

Comment:
I was expecting something like `isFalse: node.text === node.data.get('whenFalse')` which is more accurately determining when the condition is false.

Reply:
Ah yes yes this is much better!